geolocation/android/src/main/java/com/capacitorjs/plugins/geolocation/Geolocation.java

```diff
@@ -68,7 +68,7 @@ public void sendLocation(boolean enableHighAccuracy, final LocationResultCallbac
     }
 
     @SuppressWarnings("MissingPermission")
-    public void requestLocationUpdates(boolean enableHighAccuracy, int timeout, final LocationResultCallback resultCallback) {
+    public void requestLocationUpdates(boolean enableHighAccuracy, long timeout, long interval, long minInterval, final LocationResultCallback resultCallback) {
```

Comment:
Using long type seems to be better as `LocationRequest.Builder` accepts long arguments.